### DISCUSSION TITLE
Render Markdown file from different repository in Git page project repository

### DISCUSSION THREAD

**Discussion by mbabu48** [community/community#55668] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi Team, I maintain my documentation in one repository github pages. All the pages under my documentation repo uses one one of the jekyll theme "Cayman". All the markdown files within the repository are rendered perfectly. However, My documentation needs to refer other repository markdown files. Those files are not rendered rather URL is redirecting to markdown page location. Is there any way I can make them rendered and show as HTML pages in my git pages ?

example:
1. my-research-pages-repo 
- index.md
- research-areas.md

2. machine_learning_research-repo
- README.md

3. programming_language_research-repo
- READMe.md

git pages repo refers my-research-pages-repo/index.md and able to render research-areas.md. However, when I refer machine_learning_research-repo/README.md. It redirects to the markdown page instead of rendering and showing it as html page.

If this is not an existing feature. Can we request to consider as new product feature request ?

Please let me know if you need any more information.

Thanks
Naresh.

**Comment by airtower-luna** [comment]
> Is there any way I can make them rendered and show as HTML pages in my git pages ?

You could include them in your Pages site during the build. If you [publish using a custom workflow](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#publishing-with-a-custom-github-actions-workflow), you could retrieve the READMEs before running the Jekyll build, so HTML gets built just like for the files in the pages repository.

One catch: Changes to the other repositories can't directly trigger a build in your pages repository. You could set up workflows in those repositories that send [`workflow_dispatch` events](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch) to the pages repository if necessary.
